integration_test/tests/connection_tests.rs

```diff
@@ -20,6 +20,7 @@ mod integration {
     fn test_default_connection() {
         let env_handle = allocate_env();
         let conn_str = crate::common::generate_default_connection_str();
+        // let conn_str = format!("{conn_str};LOGLEVEL=debug");
```

Comment:
I left this here on purpose to demonstrate dynamic log level configuration. I still need to figure out how to test this.